transforms/code/license_check/python/README.md

```diff
@@ -0,0 +1,64 @@
+# License Check 
+
+Please see the set of
+[transform project conventions](../../../README.md#transform-project-conventions)
+for details on general project conventions, transform configuration,
+testing and IDE set up.
+
+## Summary 
+
+This filter scans the license column of an input dataset and appends the `license_status` column to the dataset.
+
+The type of the license column can be either string or list of strings. For strings, the license name is checked against the list of approved licenses. For list of strings, each license name in the list is checked against the list of approved licenses, and all must be approved.
+
+If the license is approved, the license_status column contains True; otherwise False. 
+
+## Configuration and command line Options
+
+- The optional `lc_license_column_name` parameter is used to specify the column name in the input dataset that contains the license information. The default column name is license.
+
+- The optional `lc_allow_no_license` option allows any records without a license to be accepted by the filter. If this option is not set, records without a license are rejected.
+
+- The required `lc_licenses_file` options allows a list of licenses to be specified. An S3 or local file path should be supplied (including bucket name, for example: bucket-name/path/to/licenses.json) with the file contents being a JSON list of strings. For example:
```

Comment:
this is available for the cli parser, but is not supported in the transform init().